docs/topics/cache.txt

```diff
@@ -548,6 +548,15 @@ flag on the connection's socket::
         }
     }
 
+``pymemcache`` by default sets the following options (you can override them on
+your :setting:`OPTIONS <CACHES-OPTIONS>`)::
```

Comment:
The `PyMemcacheCache` backend sets this options, not `pymemcache` itself, so maybe:
```suggestion
By default, the ``PyMemcacheCache`` backend sets the following options (you can
override them in your :setting:`OPTIONS <CACHES-OPTIONS>`)::
```